core/lib/store-assets.ts

```diff
@@ -13,7 +13,7 @@ const storeHash = process.env.BIGCOMMERCE_STORE_HASH ?? '';
  * @returns {string} The CDN image URL.
  */
 const cdnImageUrlBuilder = (sizeSegment: string, source: string, path: string): string => {
-  return `https://${buildConfig.get('urls').cdnUrl}/s-${storeHash}/images/stencil/${sizeSegment}/${source}/${path}`;
+  return `https://${buildConfig.get('urls').cdnUrls[0]}/s-${storeHash}/images/stencil/${sizeSegment}/${source}/${path}`;
```

Comment:
I think we should `at` instead of array index access and that will force us to handle the case where this array is empty.

Reply:
Do we need to handle the empty case if there is a default value being set by us?

Reply:
Changed to use `.at`

Reply:
In that case, we can type it as `[string, ...string[]]` which will then make sure we always have a default. 

Or a better option is to have the build config have a method that returns the default so that the fact we always have one is enforced by the interface we've modeled